### PR TITLE
Refactor frontend

### PR DESCRIPTION
**Pull request by danzvara**
Some frontend refactoring. Paginated the rest of the tables, moved them into separate tabs and cleaned the code a bit.

### CODE REVIEW

**Comment by danzvara** [comment]
Oh damn, commited my deploy.sh. Will fix

File: dapp/.eslintrc.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
             "error",
             "always"
         ],
-        "no-unused-vars": "warn"
+        "no-unused-vars": "warn",
+        "react/jsx-uses-vars": 1
```
Comment (fangyi-zhou) [inline dapp/.eslintrc.json]: Couldn't we get rid of the `var`s?

File: dapp/src/components/App.jsx [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        for (let c in data) { buyerContracts.push(data[c]); }
+        this.setState({
+          myBuyerContracts: buyerContracts,
+        });
+      });
+  }
+
+  buyEnergy(contractAddress) {
+    const contract = makeContract(contractAddress);
+    var index = 0;
```
Comment (fangyi-zhou) [inline dapp/src/components/App.jsx]: `var` -> `let`

File: dapp/src/components/App.jsx [hunk 3]
```diff
@@ -0,0 +1,290 @@
...
+    for (var i = 0; i < this.state.contracts.length; i++) {
```
Comment (fangyi-zhou) [inline dapp/src/components/App.jsx]: `var`

File: dapp/src/components/Column.jsx [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            </div>
+            {props.content}
+          </div>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export default Column;
```
Comment (fangyi-zhou) [inline dapp/src/components/Column.jsx]: newline

File: dapp/src/components/buyEnergyPanel.jsx [hunk 5]
```diff
@@ -5,7 +5,7 @@ import {BButton, InputBond, TransactionProgressLabel} from "parity-reactive-ui";
 const $ = require("jquery");
 $.DataTable = require("datatables.net-bs");
 
-export class AvailableContractsTable extends React.Component {
+class BuyEnergyPanel extends React.Component {
```
Comment (fangyi-zhou) [inline dapp/src/components/buyEnergyPanel.jsx]: This is inconsistent.

File: dapp/.eslintrc.json (hunk 1 above)
Comment (fangyi-zhou) [reply dapp/.eslintrc.json]: Sorry. Ignore

**Comment by fangyi-zhou** [comment]
force pushed to remove deploy

**Comment by fangyi-zhou** [comment]
@danzvara Front end is still slow, but I think this PR is good to go.

**Pull request merged**